### PR TITLE
Fix in mobile view, editor ends up with height 0 due to flexbox

### PR DESCRIPTION
**Pull request by flamerohr**
~Primarily fixes Campaign admin~
<details>

screen shot 2017-06-02 at 11 29 35 am

</details>

Ok, reinvestigated, because I thought this would be great to get in to beta1...
This fixes the Editor being hidden (zero height) in asset-admin mobile view.

The screenshot I provided to campaigns was actually a different pull request I was working on in parallel to this one.

### CODE REVIEW

**Comment by tractorcow** [comment]
Has this been tested in safari? (noted the safari specific comment)

**Comment by flamerohr** [comment]
Yes, I had checked this out in Safari (like you, because of that Safari comment) and it seems fine with or without the height rule :)

**Comment by tractorcow** [comment]
Wait, why does asset-admin PR fix campaign admin? What if you have campaign admin without asset-admin installed?

**Comment by tractorcow** [comment]
I can confirm that campaign admin without asset-admin installed is broken. do you have a missing PR somewhere?

**Comment by tractorcow** [comment]
I'll assign back to you .:)

File: client/src/containers/Editor/Editor.scss [hunk 1]
```diff
@@ -2,7 +2,6 @@
   background-color: $body-bg;
   z-index: 2;
   position: absolute;
-  height: 100%; // Safari fix for scroll, should be better fix
```
Comment (tractorcow) [inline client/src/containers/Editor/Editor.scss]: Need a fix for campaign-admin as well as asset-admin.

**Comment by flamerohr** [comment]
> Wait, why does asset-admin PR fix campaign admin? What if you have campaign admin without asset-admin installed?

This seems like a bigger story than what I'd intend this fix to be for, there's a tight coupling between campaign-admin and asset-admin in the styles... perhaps we could create a new story to investigate and fix this for beta2 instead?

**Comment by flamerohr** [comment]
@tractorcow reinvestigated this - this fixes the Editor being hidden (zero height) in asset-admin mobile view.
I think I was also fixing the campaign-admins screenshot in parallel and ended up getting that wire crossed.

**Comment by flamerohr** [comment]
I think this https://github.com/silverstripe/silverstripe-admin/pull/116 will solve both asset-admin and campaign-admin cleanly :)

**Pull request closed without merging**